conversiontracerbullet/src/test/java/org/stanford/AuthDBPropertiesTest.java

```diff
@@ -28,23 +29,24 @@
 
     private AuthDBProperties authProps;
     private Properties serverConf;
-    private String propertyFile;
+    private String serverConfResourceName = "/server.conf";
     private File serverConfFile;
     private File tmpDir;
 
     @Before
     public void setUp() throws IOException, InvocationTargetException, IllegalAccessException, NoSuchMethodException, NoSuchFieldException {
+        tmpDir = createTempDirectory("AuthDBPropertiesTest_").toFile();
         authProps = new AuthDBProperties();
-        // Use private method to get the server.conf file path
-        Field f = AuthDBProperties.class.getDeclaredField("propertyFile");
-        f.setAccessible(true);
-        propertyFile = (String) f.get(authProps);
-        serverConfFile = new File(propertyFile);
-        // Use private method to get the server.conf properties
-        Method m = AuthDBProperties.class.getDeclaredMethod("loadDataSourceProperties", String.class);
+
+        Method m = AuthDBProperties.class.getDeclaredMethod("loadPropertyResource");
```

Comment:
Am I misunderstanding this?   It seems like this line is calling the class method to get data, and then we are matching below (testConstructor, testConstructorWithString) if the data we got using the same method, but called differently (directly rather than with "getDeclaredMethod"), is equal.  I would always expect the same method to give the same result, given the same data ...

It looks like the test was that way before this PR ... just thinking it might be better to check the loaded properties against hardcoded known value strings here.